_pages/en/strategy-open-first.md

```diff
@@ -55,97 +59,100 @@ All software licensed under an [Open Source Initiative approved license](https:/
+It also indicates a commitment to actively support standards organizations and developer communities.
 
 ## 3. Commitments
 
-This section links this strategy to key GC and ESDC legislation, policies, directives, guidelines and more.
+This section links this strategy to GC and ESDC legislations, policies, directives, guidelines and more.
 
 ### Mandate Letters
 
-Mandate letters outline the objectives that each minister will work to accomplish, as well as the pressing challenges they will address in their role.
+Mandate letters outline the objectives that each minister seeks to accomplish, as well as pressing challenges to be addressed in their role.
 
-In the mandate letter for the [Minister of Digital Government](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-digital-government-mandate-letter) one of the top priorities is:
+The [Minister of Digital Government](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-digital-government-mandate-letter) mandate letter outlined the following top priorities:
 
-- Encourage the use and development of open source products and open data, allowing for experimentation within existing policy directives and building an inventory of validated and secure applications that can be used by government to share knowledge and expertise to support innovation.
+- encourage the use and development of open source products and open data, allowing for experimentation within existing policy directives and building an inventory of validated and secure applications that can be used by government to share knowledge and expertise to support innovation.
 
-The mandate letters for the [Minister of Employment, Workforce Development and Disability Inclusion](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-employment-workforce-development-and-disability-inclusion) and the [Minister of Families, Children and Social Development](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-families-children-and-social-development-mandate-letter) both mention the following:
+The [Minister of Employment, Workforce Development and Disability Inclusion](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-employment-workforce-development-and-disability-inclusion) and the [Minister of Families, Children and Social Development](https://pm.gc.ca/en/mandate-letters/2019/12/13/minister-families-children-and-social-development-mandate-letter) mandate letters outlined the following top priorities:
 
-- Commitment to govern in a positive, open and collaborative way.
-- Continue to raise the bar on openness, effectiveness and transparency in government.
-- This means a government that is open by default.
-- It means better digital capacity and services for Canadians.
+- commitment to govern in a positive, open and collaborative way;
+- continue to raise the bar on openness, effectiveness and transparency in government;
+  - this means a government that is open by default; and
```

Comment:
I think the indent at lines 142 and 143 isn't voluntary?

Reply:
I think its ok like this.

In the mandate letters it's 2 separate "priorities"

> That starts with a commitment to govern in a positive, open and collaborative way.

and

> I also expect us to continue to raise the bar on openness, effectiveness and transparency in government.
This means a government that is open by default.
It means better digital capacity and services for Canadians.